controls/slick.pager.js

```diff
@@ -75,8 +98,29 @@
       var $settings = $("<span class='slick-pager-settings' />").appendTo($container);
       $status = $("<span class='slick-pager-status' />").appendTo($container);
 
+      var pagingOptions = '';
+
+      for (var o = 0; o < _options.pagingOptions.length; o++)
+      {
+        var p = _options.pagingOptions[o];
+
+        pagingOptions += "<a data=" +p.data + ">" + p.name + "</a>";
+      }
+      console.log(pagingOptions);
```

Comment:
the `console.log` was probably put for debugging but shouldn't be merged

Reply:
true, realized this myself, too, will remove asap...